code/datums/observation/observation.dm

```diff
@@ -62,76 +62,241 @@
 /decl/observ
 	var/name = "Unnamed Event" // The name of this event, used mainly for debug/VV purposes. The list of event managers can be reached through the "Debug Controller" verb, selecting the "Observation" entry.
 	var/expected_type = /datum // The expected event source for this event. register() will CRASH() if it receives an unexpected type.
-	var/list/event_sources     // Associate list of event sources, each with their own associate list. This associate list contains an instance/proc pair to call when the event is raised.
+	var/list/event_sources     // Associative list of event sources, each with their own associative list. This associative list contains an instance/proc pair to call when the event is raised.
 	var/list/global_listeners  // Associative list of instances that listen to all events of this type (as opposed to events belonging to a specific source) and the proc to call.
 
 /decl/observ/New()
 	all_observable_events.events += src
-	event_sources = list()
-	..()
+	return ..()
```

Comment:
Actually, ref say one thing and informal posts another. 
Go with the ref then, being more official. If something comes up it means we can make a BYOND issue ticket.